Locking Updates (#579)
hashicorp/terraform-website: content/source/docs/enterprise/workspaces/settings.html.md

If you need to prevent Terraform runs for any reason, you can lock a workspace. This prevents users with write access from manually queueing runs, prevents automatic runs due to changes to the backing VCS repo, and prevents the creation of runs via the API. To enable runs again, a user must unlock the workspace.

> Users with write access are permitted to lock and unlock a workspace. However, they can't unlock a workspace which was locked by another user. Users with admin privileges can force unlock a workspace even if another user has locked it.

Reviewer: ```suggestion
Users with write access can lock and unlock a workspace, but can't unlock a workspace which was locked by another user. Users with admin privileges can force unlock a workspace even if another user has locked it.
```

> Users with write access are permitted to lock an unlocked workspace; however, they are not allowed to unlock a workspace if another user locked it. Users with admin privileges are able to force unlock a workspace if another user has locked it.

Reviewer 2: This is correct as-is, but it's a bit subtle. I tried to clarify and simplify the language a bit:

```suggestion
Users with write access are permitted to lock and unlock a workspace. However, they can't unlock a workspace which was locked by another user. Users with admin privileges can force unlock a workspace even if another user has locked it.
```

Author: yeah, i totally struggled with writing this in a clear way. Yours is better.

> Locks are managed with a single "Lock/Unlock `<WORKSPACE NAME>`" button. TFE asks for confirmation when unlocking.

Reviewer: ```suggestion
Locks are managed with a single "Lock/Unlock/Force unlock `<WORKSPACE NAME>`" button. TFE asks for confirmation when unlocking.
```
